src/routes/Privacy.js

```diff
@@ -0,0 +1,125 @@
+import React, { Component } from 'react'
+
+class Privacy extends Component {
+  render() {
+    return (
+      <div className="privacy">
```

Comment:
Should not be using string class names. Please used styled-components instead.

Reply:
I actually copied and pasted the one Jeff originally created which had these classname (it's used in other places as well) so I just followed the existing convention. Can we make styled component bits out of scope of this review, as my focus is to put the content in, not about styling?